pkg/placement/placement-resolution.go

```diff
@@ -0,0 +1,363 @@
+/*
+Copyright 2023 The KubeStellar Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package placement
+
+import (
+	"fmt"
+	"sync"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/types"
+
+	"github.com/kubestellar/kubestellar/api/edge/v1alpha1"
+	"github.com/kubestellar/kubestellar/pkg/util"
+)
+
+const clusterScopedNamespace = ""
```

Comment:
Why not use https://github.com/kubernetes/apimachinery/blob/v0.28.2/pkg/apis/meta/v1/types.go#L280 ?

Reply:
@MikeSpreitzer 
All comments until this point were implemented (unless I left a question in comment) in commit [2d3373040a3530d9e4e2f491cc807a68c34cf43f](https://github.com/kubestellar/kubestellar/pull/1681/commits/2d3373040a3530d9e4e2f491cc807a68c34cf43f).

All comments below this point (even if now marked outdated) - are not addressed yet.